ton_client/client/src/tvm/check_transaction.rs

```diff
@@ -19,6 +19,7 @@ use std::convert::TryFrom;
 pub(crate) async fn check_transaction<F>(
     transaction: &ton_block::Transaction,
     real_tr: bool,

```

Comment:
What does it mean `real` ?
As I understand, this parameter must be named as `include_transaction_id_in_error`

Reply:
Real means that it is a transaction received from blockchain in opposition to transaction created by local executor. The function can do additional fetures regulated by this flag in future so I keep the name